src/screens/GameDetails/index.tsx

```diff
@@ -0,0 +1,138 @@
+import React, { useState, useEffect } from 'react';
+import { StyleSheet, useWindowDimensions } from 'react-native';
+import {
+  AspectRatio,
+  VStack as NativeBaseVStack,
+  Text,
+  HStack,
+  Fab,
+  useToast,
+  useTheme,
+} from 'native-base';
+import { useRoute } from '@react-navigation/native';
+import FastImage from 'react-native-fast-image';
+import RenderHtml from 'react-native-render-html';
+import { FloppyDisk } from 'phosphor-react-native';
+
+import ScrollView from '@components/ScrollView';
+import Header from '@components/Header';
+import ScreenWrapper from '@components/ScreenWrapper';
+import VStack from '@components/VStack';
+import Loading from '@components/Loading';
+import { Game } from '@interfaces/game.dto';
+import rawg from '@services/rawg.api';
+import { useAppDispatch } from '@src/store';
+import { setDrawerHeader } from '@store/slices/navigation-slice';
+import { GAMEAPI_KEY } from 'react-native-dotenv';
+
+type RouteParams = {
+  id: number;
+  slug: string;
+  name: string;
+};
+
+const GameDetails = () => {
+  const toast = useToast();
+  const { colors } = useTheme();
+  const dispatch = useAppDispatch();
+  const route = useRoute();
+  const { id, name } = route.params as RouteParams;
+  const { width } = useWindowDimensions();
+  const [game, setGame] = useState<Game>({} as Game);
+  const [isLoading, setIsLoading] = useState<boolean>(true);
+
+  useEffect(() => {
+    const getGame = async () => {
+      try {
+        const response = await rawg.get<Game>(`games/${id}?key=${GAMEAPI_KEY}`);
+
+        setGame(response.data);
+      } catch (err) {
+      } finally {
+        setIsLoading(false);
+      }
```

Comment:
Can we add some kind of logger to this catch statement? I'm not sure what is good practice in RN but as you're already handling the error we could add this for a debug logger.